src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -565,6 +565,37 @@ def test_traffic_routing(self, resource_group):
         self.cmd('webapp traffic-routing clear -g {} -n {}'.format(resource_group, webapp))
 
 
+class AppServiceCors(ScenarioTest):
+    @ResourceGroupPreparer()
+    def test_webapp_cors(self, resource_group):
+        self.kwargs.update({
+            'plan': self.create_random_name(prefix='slot-traffic-plan', length=24),
+            'web': self.create_random_name(prefix='slot-traffic-web', length=24),
+        })
+        self.cmd('appservice plan create -g {rg} -n {plan} --sku S1')
+        self.cmd('webapp create -g {rg} -n {web} --plan {plan}')
+        self.cmd('webapp cors add -g {rg} -n {web} --allowed-origins https://msdn.com https://msn.com')
```

Comment:
once we add support for slots can you add another test here as well for slots? Thanks!

Reply:
done